MIGRATION.md

```diff
@@ -18,8 +18,8 @@ To use the script:
   ```javascript
   // your-custom-language.json
   {
-    "locale": "en_US",
-    "phrases": {
+    "locale": "en_US",  // untouched keys
```

Comment:
Line 09: 
From version `6.x.y`, Web SDK will use a new locale key naming convention that better supports scalability.

Line 10: 
As a result, many key names will be changed and this might affect the integrator's custom locale options.

Line 10-11: 
The migrate_locales script will help integrators migrate from the older key name convention to the new one with minimal hassle.